README.md

```diff
@@ -64,3 +64,19 @@ Additionally, there a number of secret variables (only required in production) d
 - **SECRET_KEY_BASE**: Used for writing encrypted cookies. Generate a value using `mix phx.gen.secret` (only required in production)
 - **SWIFTLY_AUTHORIZATION_KEY**: Authorization key for Swiftly
 - **BRIDGE_API_USERNAME**, **BRIDGE_API_PASSWORD**: credentials for the API that gets drawbridge status
+
+## Production Deploys
+
+Prior to releasing a new version to prod, we create an annotated Git tag documenting what's being deployed. The tag name should be in the format `YYYY-MM-DD-N1, where `N` is used to differentiate between multiple releases in a given day and starts at 1. For the body of the description, run the following command:
+```
+git log --abbrev-commit --pretty=format:'%h:%s' [hash of previous prod version]..HEAD
+```
+Then run:
+```
+git tag -a [tag name]
+```
+which will prompt you to enter the annotation from the previous step via your editor. Once this is done, you can push using:
+```
+git push origin --tags
+```
+Once this is complete, initiate the deploy through AWS and post a link to the tag on GitHub in the `#ctd-deploys` Slack channel.
```

Comment:
Do we need more explicit instructions here?

Reply:
I was sort of on the fence here just because the current Elastic Beanstalk deploy process is about to go away, to be replaced with something in ECS that we don't yet understand quite fully enough to document. I'd say that once we have a better understanding of the ECS process we should definitely document that.